### ISSUE TITLE
MuseTree not working

### ISSUE THREAD

**Issue by AndrianaMantidfly** [stevenwaterman/musetree#113]
I don't know what happened, but for some reason it hasn't been working for a week somewhere.. I'm trying to make a request, but then it does nothing.. Did something happen?

**Comment by Kalto-Mate** [comment]
To add some context for this, what happens right now is you left click a node to request more as usual, and the "+4" appears without issues. After a short while comparable to the usual time a request takes to be fulfilled, however, the "+4" disappears but no new leaf nodes are added to the parent node you requested more for. Either the request isn't making it to Musenet, or an API change is making Musetree fail to fetch and/or display the results.

**Comment by AndrianaMantidfly** [comment]
Nothing happens when I ask. I tried to use MuseNet and nothing happens there except for the demo version.

**Comment by stevenwaterman** [comment]
Yeah unfortunately MuseNet itself has been down for a couple of weeks. Since MuseTree is just making calls to the MuseNet api behind the scenes. there's not really anything I can do other than hope MuseNet comes back online

**Comment by Kalto-Mate** [comment]
Dang, This is a very useful tool to brainstorm with, I hope it doesn't come to one having to host their own Musenet. 
(Is it even open source at that level?)

**Comment by stevenwaterman** [comment]
No, it's a completely closed source model, and they never commercialised it or anything. There are other music generation models coming out, but it would be a case of having to run it locally

**Comment by abrosen** [comment]
That's too bad.  I found MuseNet after Google's update to colabs broke their own music transformer colab notebook last year;  now MuseNet is dead too.  AI music can't seem to stay alive.

**Comment by andresonca** [comment]
any news about it?

**Comment by Fitsbit** [comment]
I started to use Musetree and it seems great, the orignal MuseNet seems to be updated so there could be a fix soon.

**Comment by Kalto-Mate** [comment]
> I started to use Musetree and it seems great, the orignal MuseNet seems to be updated so there could be a fix soon.

Don't give me false hope :(

**Comment by Fitsbit** [comment]
no Musenetis back up, check it out.  https://openai.com/research/musenet

**Comment by abrosen** [comment]
> no Musenetis back up, check it out. https://openai.com/research/musenet

That particular URL never went down.  However, the advanced settings are still missing.  I would be ecstatic to be wrong.

**Comment by Kalto-Mate** [comment]
Yeah see? False hope. As abrosen said, the page never went down, they just silently closed the Api up and replaced what is presented on the page, from actual generation to a preloaded set of results that are always the same. 
I hope we haven't lost this amazing tool forever just because the stink artists made about Ai art way back when SOMEHOW scared OpenAi into playing it unnecessarily extra safe around the music industry.

**Comment by asigalov61** [comment]
Hey guys!

I am posting it here for anyone who may be interested...

I was able to develop an open-source MuseNet alternative:

https://github.com/asigalov61/Los-Angeles-Music-Composer

If anyone wants to help me to write a proper API and also adapt MuseTree for it, I would really appreciate it!

Please let me know here if anyone is interested and/or can help.

Thank you.

PS. MuseNet (API) had been down for some time because they most likely re-allocated all resources to ChatGPT. It was also a freebee/passion project of OpenAI (think close-source) so it is not surprising...

**Comment by AndrianaMantidfly** [comment]
A new neural network has recently appeared and I have a crazy idea, but maybe it's worth temporarily transferring the work of this interface to it in some way?
https://crfm.stanford.edu/2023/06/16/anticipatory-music-transformer.html?idx=1#prompted-example

**Comment by asigalov61** [comment]
@SussyBoba Nice find. Thanks. I will write to them and ask them if they would be interested in collabing/making a pub API.

Meanwhile, check out my latest music model. I was able to make a simple API for it and it does the same things as the one you mentioned.

https://github.com/asigalov61/Euterpe-X

**Comment by cloph-dsp** [comment]
still nothing?
